contrib/babelfishpg_common/sql/smalldatetime.sql

```diff
@@ -482,7 +487,13 @@ DEFAULT FOR TYPE sys.SMALLDATETIME USING btree AS
     OPERATOR    3   =  (sys.SMALLDATETIME, sys.SMALLDATETIME),
     OPERATOR    4   >= (sys.SMALLDATETIME, sys.SMALLDATETIME),
     OPERATOR    5   >  (sys.SMALLDATETIME, sys.SMALLDATETIME),
-    FUNCTION    1   smalldatetime_cmp(sys.SMALLDATETIME, sys.SMALLDATETIME);
+    FUNCTION    1   sys.smalldatetime_cmp(sys.SMALLDATETIME, sys.SMALLDATETIME),
+    OPERATOR    1   <  (sys.SMALLDATETIME, date),
+    OPERATOR    2   <= (sys.SMALLDATETIME, date),
+    OPERATOR    3   =  (sys.SMALLDATETIME, date),
+    OPERATOR    4   >= (sys.SMALLDATETIME, date),
+    OPERATOR    5  >  (sys.SMALLDATETIME, date),
+    FUNCTION    1   sys.smalldatetime_date_cmp(sys.SMALLDATETIME, date);
```

Comment:
Instead of modifying the existing operator class, I would suggest to create new operator class named like `sys.smalldatetime_date_ops`.

Reply:
Created a new operator class that adds operator to smalldatetime_ops family.